apps/generic/ui/aboutdialog.cpp

```diff
@@ -61,7 +61,7 @@ AboutDialog::AboutDialog(QWidget *parent) :
     vlayout->addWidget(label1);
 
     QLabel *label2 = new QLabel(tr("Thanks to") + "\n" \
-                                "XBMC Development Team\n");
+                                "Kodi Development Team\n");
```

Comment:
I think their official name is "Team XBMC" (also now with the Kodi naming). So might be a good time to update this (I think this line is only present in the generic build, or at least not in the Sailfish one AFAIK).